src/lib.rs

```diff
@@ -523,7 +523,6 @@ impl<M: Manager> Pool<M> {
         let create_r = self.open_new_connection().await;
 
         if create_r.is_ok() {
-            decrement_gauge!(IDLE_CONNECTIONS, 1.0);
```

Comment:
Do we not need a `decrement` _somewhere_? Otherwise this is more a counter that can only go up.

Reply:
The diff doesn't show it, but there is a decrement on line 515 when we use an idle connection. 

https://github.com/importcjj/mobc/pull/86/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759L515